web/src/components/loadingplaceholder.jsx

```diff
@@ -1,8 +1,8 @@
 import styled from 'styled-components';
 
 export default styled.div`
-  background: transparent url(${resolvePath('images/electricitymap-icon.svg')}) no-repeat center center;
-  background-size: 2rem;
+  background: transparent url(${resolvePath('images/electricitymap-loading.svg')}) no-repeat center center;
+  background-size: 10rem;
```

Comment:
```suggestion
  background: transparent url(${resolvePath('images/electricitymap-loading-icon.svg')}) no-repeat center center;
  background-size: 10rem;
```
Are the background-size changes needed?

Reply:
No those would be breaking, I've fixed it another commit

Reply:
<img width="382" alt="image" src="https://user-images.githubusercontent.com/45588799/184835603-bbb03bbf-eea8-4c69-8f13-980cab5fd2c1.png">